<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IFavourites</class>
 <widget class="QDialog" name="IFavourites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel favourites</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/toolicons/gfx/main_bookmarks.png</normaloff>:/toolicons/gfx/main_bookmarks.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="edChannel"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButton">
     <property name="toolTip">
      <string>Add channel to list</string>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/gfx/add.png</normaloff>:/icons/gfx/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnJoin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Join</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListView" name="chanView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkJoin">
     <property name="text">
      <string>Join on connect</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Channel key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="edKey">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="btnDelete">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edChannel</tabstop>
  <tabstop>btnJoin</tabstop>
  <tabstop>chanView</tabstop>
  <tabstop>chkJoin</tabstop>
  <tabstop>edKey</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
